src/library/browse/browsefeature.cpp

```diff
@@ -322,6 +353,7 @@ void BrowseFeature::onLazyChildExpandation(const QModelIndex& index) {
 
     // If we are on the special device node
     if (path == DEVICE_NODE) {
+#if defined(__WINDOWS__)
         // Repopulate drive list
         QFileInfoList drives = QDir::drives();
```

Comment:
The code for windows could also be extracted into a `getRemovableDevices()` function. Finally you can simplify the code as follows:

```
#if defined(__LINUX__) || (__WINDOWS__)
folders += getRemovableDevices(this);
#else
DEBUG_ASSERT(!"Removable devices are not supported for this platform");
#endif

What do you think?

Reply:
Done.